miasm2/analysis/influencegraph.py

```diff
@@ -0,0 +1,293 @@
+"""Provide influence graph"""
+
+import miasm2.expression.expression as m2_expr
+from miasm2.analysis.depgraph import DependencyNode, DependencyDict, \
+    DependencyResult, DependencyResultImplicit, FollowExpr, DependencyGraph
+from miasm2.expression.simplifications import expr_simp
+from miasm2.core.asmbloc import expr_is_label
+from miasm2.core.graph import DiGraph
+
+import itertools
```

Comment:
For more homogeneity, please first import Python module as `itertools` and `collections`, and then import miasm2 relative modules (take a look at [PEP08](https://www.python.org/dev/peps/pep-0008/#imports) for details).